### ISSUE TITLE
Bind-mount Docker socket into containers

### ISSUE THREAD

**Issue by mpdude** [JonathonReinhart/scuba#141]
In some cases, it may be helpful if containers are allowed to run additional `docker` commands or possibly start additional containers. This can be achieved by mounting the Docker socket into containers.

It would be nice if we could have an option to do so per-alias. 

For security reasons, this should *not* be the default behavior, since to my understanding this gives the running container root access to the host machine.

**Comment by xanarin** [comment]
@JonathonReinhart I believe this feature is now possible with the addition of the `docker_args:` directive in #177, so this issue should be closed.

**Comment by JonathonReinhart** [comment]
FYI I just verified that that with #186 and the following `.scuba.yml`:
```yaml
image: docker:latest
volumes:
  /var/run/docker.sock: /var/run/docker.sock
aliases:
  docker:
    root: true
    script: docker
```

I can successfully run docker from inside of scuba:
```
$ docker info | grep ID:
 ID: 4QVQ:QUFW:MNEK:QZMW:GOLM:MFBH:LXMV:GCSH:OV4F:PIAS:QDEC:3NTG
$ scuba docker info | grep ID:
 ID: 4QVQ:QUFW:MNEK:QZMW:GOLM:MFBH:LXMV:GCSH:OV4F:PIAS:QDEC:3NTG
```

**Comment by JonathonReinhart** [comment]
I just released [Scuba v2.9.0](https://github.com/JonathonReinhart/scuba/releases/tag/v2.9.0) which adds support for `volumes`, as shown above, so I will consider this issue solved. Please let me know if there's anything else!
